From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Wed, 28 Aug 2013 18:43:42 +0200
Subject: creating a dummy .deps/ directory to get make happy

The Mozilla buildsystem doesn't take care on the directory
  mozilla/js/src/.deps/
so we added this to let make/g++ doesn't stop.
---
 mozilla/js/src/.deps/dummy | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 mozilla/js/src/.deps/dummy

diff --git a/mozilla/js/src/.deps/dummy b/mozilla/js/src/.deps/dummy
new file mode 100644
index 0000000..d5f14fe
--- /dev/null
+++ b/mozilla/js/src/.deps/dummy
@@ -0,0 +1 @@
+dummy file to make git happy
